README.md

```diff
@@ -243,6 +244,28 @@ See gif in action:
 The built-in multi-cursor support may not meet your needs. Please refer to the plugin
 [vscode-multi-cursor.nvim](https://github.com/vscode-neovim/vscode-multi-cursor.nvim) for more multi-cursor features
 
+### Remote development
+
+We intend to use vscode-neovim as a UI extension, running locally instead of installing and configuring nvim in a remote
```

Comment:
can we instead declare `["ui", "workspace"]` and try to do the right thing (and possibly show a message) automatically?

Reply:
There should be no way to determine where the user wants to install the extension.